packages/hoppscotch-app/components/lenses/renderers/composables/useCopyResponse.ts

```diff
@@ -0,0 +1,22 @@
+import { Ref, ref } from "@nuxtjs/composition-api"
+import { copyToClipboard } from "~/helpers/utils/clipboard"
+
+export default function useCopyResponse(
+  responseBodyText: Ref<any>,
+  toast: any,
+  t: any
+): { [key: string]: (() => void) | Ref<any> } {
```

Comment:
The return type defined is too broad. Please define a more concrete return type. If you have difficulties figuring out the narrorw return type, you can just remove the return type and TypeScript should give you a hint with the more correct return type.

Reply:
Great suggestion!